paasta_tools/cli/schemas/kubernetes_schema.json

```diff
@@ -780,6 +780,11 @@
                     "type": "integer",
                     "minimum": 1
                 }
+            },
+            "dependentRequired": {
+                "min_instances": [
+                    "max_instances"
+                ]
```

Comment:
do you also want the inverse relationship? i.e.

```
"max_instances": [
    "min_instances"
]
```

Also, if you've got this constraint specified in the schema, do you even need to write code that checks for the same thing?

Reply:
should be able to write a test that calls the validate function that uses the schema (there should. be some tests in https://github.com/Yelp/paasta/blob/master/tests/cli/test_cmds_validate.py that do this)

Reply:
Added tests for this